tools/check-templates

```diff
@@ -44,10 +52,45 @@ def check_html_templates(templates, modified_only):
     # there are also cases where Casper deliberately uses invalid HTML,
     # so we exclude them from our linter.
     templates = filter(
-        lambda fn: ('base.html' not in fn) and ('casperjs' not in fn),
+        lambda fn: (('base.html' not in fn) and ('casperjs' not in fn)),
         templates)
     templates = sorted(list(templates))
 
+    template_id_dict = build_id_dict(templates)
+    # TODO: Clean up these cases of duplicate ids in the code
+    IGNORE_IDS = [
+        'api-example-tabs',
+        'errors',
+        'email',
+        'registration',
+        'pw_strength',
+        'id_password',
+        'top_navbar',
+        'id_email',
+        'id_terms',
+        'send_confirm',
+        'register',
+    ]
+    ignorable_ids_dict = {ids: fns for ids, fns in iteritems(template_id_dict)
+                          if ids in IGNORE_IDS and len(fns) > 1}
+    bad_ids_dict = {ids: fns for ids, fns in iteritems(template_id_dict)
+                    if (not ids in IGNORE_IDS) and len(fns) > 1}
+
+    for ids, fns in iteritems(ignorable_ids_dict):
```

Comment:
We usually say `foo.items()` instead of `iteritems(foo)`.  In Python2, `iteritems` would occasionally have performance advantages, but only for really long lists, but in Python3, I think items() is already an iterator.  So our convention here is `foo.items()`.